.github/workflows/build-check-install.yaml

```
@@ -176,6 +194,22 @@ jobs:
               writeLines(readLines(check_log))
               stop("⚠ R CMD check has warnings")
             }
+            if ("${{ inputs.check-note-blocklist }}" == "true") {
+              print("Checking notes...")
+              regexes <- "${{ inputs.note-blocklist }}"
+              regexes <- unlist(strsplit(regexes, split = "\n"))
+              lines = paste(readLines(check_log), collapse = " ")
+              notes_result <- lapply(regexes, function(r){
+                if(grepl(r, lines)){
+                  print(r)
+                  return(TRUE)
+                }
+                return(FALSE)
+              })
+              if(any(sapply(notes_result, function(notes_result) notes_result))){
```

if converted to a vector - then you can simplify it into:
```suggestion
              if (any(notes_result)) {
```